### ISSUE TITLE
main does not type-check: madUpload.test.ts references an undeclared BYTES

### ISSUE THREAD

**Issue by koeylp** [yuthstyle88/108jobs-clean#94]
`main` does not type-check and has a failing test. Both come from the same
missing constant.

## Evidence

`src/services/media/madUpload.test.ts:492`

```ts
stubFetch([SESSION, BYTES, COMPLETE]);
```

`SESSION` and `COMPLETE` are declared at lines 155-156. **`BYTES` is never
declared anywhere in the file** — it appears exactly once, at that usage.

On a clean `origin/main`:

```
tsc --noEmit   exit 2
  src/services/media/madUpload.test.ts(492,25): error TS2304: Cannot find name 'BYTES'.
pnpm test:unit exit 1
  1 failed | 439 passed (440)  —  "uploads via MAD when gateway URL is configured"
```

Introduced by 37c2070 `feat(media): integrate uploadSelectedImage with MAD and
fallback`.

## Also from the same sync

`src/app/[lang]/admin/manage-riders/page.tsx:3` imports `Image` from
`next/image` and never uses it — the file uses `AvatarImage`, a different
symbol. It lints as `'Image' is defined but never used`, taking the repo's
warning floor from 601 to 602.

## Why this is being reported rather than fixed

The fix needs a stub for the byte-upload step of the MAD flow, matching what
`uploadSelectedImage` expects between the session and complete calls. That is
domain knowledge belonging to whoever wrote the feature; inventing a fixture
would produce a test that passes without proving anything.

Found while merging `origin/main` into a rider-admin branch — the failure is not
that branch's and reproduces on `main` alone.
